docs/install.rst

```diff
@@ -2,6 +2,8 @@
 Installation
 ************
 
+.. _installing-astropy:
```

Comment:
Do add the new dependencies to this docs page, too, links to optional ones start on L157

Reply:
Good point.  Will do!

Reply:
Resolved this comment by adding the new dependencies to `install.rst` via 33e9b2f641815cc01faf635d66df866690501f8d.

Thank you! :+1: